products/haproxy.md

```diff
@@ -22,13 +22,13 @@ releases:
 -   releaseCycle: "2.6"
     lts: true
     eol: 2027-04-01
-    latest: "2.6.7"
-    latestReleaseDate: 2022-12-02
+    latest: "2.6.8"
+    latestReleaseDate: 2023-01-24
     releaseDate: 2022-05-31
 -   releaseCycle: "2.5"
     eol: 2023-01-01
-    latest: "2.5.10"
-    latestReleaseDate: 2022-12-05
+    latest: "2.5.11"
```

Comment:
Note: 2.5 is EOL on endoflife.date (since 01 Jan 2023).

Reply:
haproxy 2.5 is still marked as maintained on https://www.haproxy.org/.

According to the product description, odd branches EOL date is not decided in advance and should be maintained from 12 to 18 months, so EOL should be between May 23, 2022-12-23 and 2023-05-23. Should we take a conservative approach and choose the latest date ? @captn3m0 ?

Reply:
#2416